pucauto.py

```diff
@@ -10,6 +12,10 @@
 
 with open("config.json") as config:
     CONFIG = json.load(config)
+    logging.config.dictConfig(CONFIG.get('logging', {
+        # sane defaults here?
+    }))
```

Comment:
I want to hide it away in a file no one will ever see. Seems like something you set once and forget, which makes me sad that I need to set it in the first place.